examples/builder_example.php

```diff
@@ -9,6 +9,8 @@
 use Sourceability\StringMatcher\Simplifier\ChainSimplifier;
 use Sourceability\StringMatcher\Simplifier\TrimmedLowercaseSimplifier;
 use Sourceability\StringMatcher\ValuesMatcherBuilder;
+use const PHP_EOL;
+use function print_r;
```

Comment:
<img width="757" alt="Screen Shot 2021-02-10 at 9 50 26 AM" src="https://user-images.githubusercontent.com/1288852/107550076-72749f00-6b85-11eb-9f9b-f787b9b641c0.png">

No, we should not remove these use statements. These were added because the examples are using those in the global namespace. And the examples are CS checked.